src/section-list.js

```diff
@@ -79,7 +84,7 @@ export default class extends Component<Props> {
     );
 
     if (
-      y <= this._startThreshold && // nearing the top
+      y <= this.props.onStartReachedThreshold && // nearing the top
```

Comment:
So wait, is this supposed to be a pt value or a ratio of the view height?

Reply:
The source for SectionList says onEndReachedThreshold should be a ratio https://github.com/facebook/react-native/blob/master/Libraries/Lists/SectionList.js#L148

Reply:
Hmm I think `y` offset in `onScroll` call returns the value in points, we're comparing it with threshold value and event gets fired X pt's before start. 

Please correct me if I'm wrong, but within your implementation, you're doing:

```javascript
const { height } = e.nativeEvent.layout; // Height of an list

const threshold = onStartReachedThreshold ? onStartReachedThreshold : 0;
this._startThreshold = height * threshold;
```

Assuming that `onStartReachedThreshold={100}` and height of a list is `100` pt your `__startThreshold` equals 10000, and it's compared to `y` value that is always lower.

Am I missing something?

Reply:
Yeah, if it was passed a large number like that, for sure. When running an example I made, I was passing it values like `0.5` for half the viewheight from the top of the list. 

The `y` offset in `onScroll` does return the value in points, but that's what the `__startThreshold` was saving.

Reply:
I see but I think `onEndReachedThreshold` works the way I've changed it:

https://facebook.github.io/react-native/docs/listview.html#onendreachedthreshold

> Threshold in pixels (virtual, not physical) for calling onEndReached.